### ISSUE TITLE
Remove Google plus

### ISSUE THREAD

**Issue by cladvi** [FortAwesome/Font-Awesome#17839]
Hi guys can you guys remove the Google plus from whole icon list and add new social platforms that not support, Google + gone forever. Now softwares use your API shows google+ we ask them to remove, we dont need it but not lookalike  they can remove, unless you removed it..if you guys remove it from your data base then all gone  Thanks you!

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

We were discussing this, but I don't know if this will happen. Google plus is not the only dead brand (ADN is another). We may want to preserve some icons for historical reasons

@sensibleworld any thoughts?

**Comment by cladvi** [comment]
> We were discussing this, but I don't know if this will happen. Google plus is not the only dead brand (ADN is another). We may want to preserve some icons for historical reasons

> @sensibleworld any thoughts?

That's Fair enough, but then it should not mix with active icons right? At least create category that put all the historical reason Icon on Legacy folder... in way not mix with active, specially when SAAS use your API... @tagliala

**Comment by tagliala** [comment]
>  specially when SAAS use your API

If I got this correctly, you are using a third party library which is using Font Awesome.

This third-party library shows you the Google+ icon, but you don't want to see it because it is outdated. Is it correct?

**Comment by cladvi** [comment]
@tagliala 
Yes something like that, we use a software that use font Awesome icons, ( can not reveal name of the software) so we ask them to remove those out dates icons, but its look like it out of their control or something... they not took any action, so we thought if **we go the origin** and ask them **a solution,** that way it may not show up on icon selection and have opportunity to add new icons..like WhatsApp, Messenger clubhouse, ticktok..what ever..... hope you guys can do something.. and if you want to keep historic icons you guys can keep it  somewhere that not interfere with active icons.... you guys awesome and smart so you know what to do.. we just suggest.....

**Comment by tagliala** [comment]
Thanks for the clarification

Since google-plus is not the only dead brand, I'm asking if [fa-adn](https://fontawesome.com/icons/adn?style=brands) is also a problem for your use case, as well as any other brand (and maybe non-brand) icon that you don't need

**Comment by cladvi** [comment]
@tagliala 
We just checked..yes **fa-adn also there**, but in our use case,we never come across Adn icon or never used it... its long gone..we never even knew it,,But Google+ was big, so once it gone we thought Font awesome icon set should need fresh new icons sets that are missing in our SAAS software we use,,but developers not aware of it..so there a bunch of icons needs to be go legacy icons... i must say :)
but i m sure other people will think same like me... if they need to be more efficient

**Comment by sensibleworld** [comment]
We're still reviewing internally the best way to deal with these. Stay tuned!
